exo-go/src/cmd/clean.go

```diff
@@ -23,56 +21,19 @@ var cleanCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		err = removeImages(c)
+		err = dockerHelpers.RemoveDanglingImages(c)
```

Comment:
Okay. These changes could have been done separately. No need to change it here now, just pointing out opportunities to trim down this PR.